_format/1.1/index.md

```diff
@@ -241,11 +241,13 @@ with a `415 Unsupported Media Type` status code.
   `profile` media type parameter is present.
 
 If a request's `Accept` header contains an instance of the JSON:API media type,
-servers **MUST** respond with a `406 Not Acceptable` status code if all
-instances of that media type are modified with a media type parameter other
-than `ext` or `profile`. If every instance of that media type is modified by the
-`ext` parameter and each contains at least one unsupported extension URI, the
-server **MUST** also respond with a `406 Not Acceptable`.
+servers **MUST** ignore instances of that media type are modified with a media
```

Comment:
servers **MUST** ignore instances of that media type **which** are modified with a media

Reply:
Thanks a lot for catching that one!

```suggestion
servers **MUST** ignore instances of that media type which are modified with a media
```